src/main/java/com/revature/models/Wishlist_products.java

```diff
@@ -0,0 +1,24 @@
+package com.revature.models;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@Entity
+@NoArgsConstructor
+@AllArgsConstructor
+public class Wishlist_products {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private int id;
+    private int wishlist_id;
+    private int product_id;
```

Comment:
Same as with the Wishlist class. Use the Wishlist and Product objects here instead of their ids and define the relationship using @ManyToOne annotations.